nav2_msgs/action/ComputeControl.action

```diff
@@ -1,5 +1,6 @@
 #goal definition
```

Comment:
I think `ComputeControl` doesn't fully describe this action -- not only computing the control signal but also actuating. Maybe `MoveBase`?

Reply:
So using the phrase `MoveBase` I think is automatically not ideal since that means something in ROS1 and seeing a `MoveBase.action` is going to be awfully confusing. 

My feeling is @mkhansen-intel will have the same arguments for this as with `ComputePlan` so these are just going to get changed back to `FollowPath`

Reply:
Good so now we are restricting this to mean only FollowPath actions -- some comments below were directed towards clarifying what type of control

Reply:
@orduno ok, can you close those comments so I can only focus on the ones assuming that all `ComputeControl`s are now `FollowPath`s?